src/styles/themes/default.js

```diff
@@ -23,7 +23,6 @@ const darkTheme = {
     danger: colors.red,
     dangerHover: colors.redHover,
     dangerPressed: colors.redHover,
-    dangerDisabled: colors.redDisabled,
```

Comment:
This is not used anywhere else, removing as it could lead to wrongful assumptions